modules/gui/ev_selection_window.py

```diff
@@ -0,0 +1,72 @@
+import time
+from tkinter import *
+
+from rich.prompt import IntPrompt
+
+from modules.context import context
+from modules.pokemon import Pokemon
+from modules.pokemon import get_party, StatsValues
+
+
+def ask_for_ev_targets(pokemon: "Pokemon") -> StatsValues:
+    if context.gui.is_headless:
+        return StatsValues(
+            hp=IntPrompt.ask("Choose target HP EVs", default=pokemon.evs.hp),
+            attack=IntPrompt.ask("Choose target Attack EVs", default=pokemon.evs.attack),
+            defence=IntPrompt.ask("Choose target Defence EVs", default=pokemon.evs.defence),
+            speed=IntPrompt.ask("Choose target Speed EVs", default=pokemon.evs.speed),
+            special_attack=IntPrompt.ask("Choose target Special Attack EVs", default=pokemon.evs.special_attack),
+            special_defence=IntPrompt.ask("Choose target Special Defence EVs", default=pokemon.evs.special_defence),
+        )
+
+    spinboxes: list[Spinbox] = []
+    selected_ev_targets: StatsValues | None = None
+
+    def remove_window(event=None):
+        nonlocal window
+        window.destroy()
+        window = None
+
+    def return_selection():

```

Comment:
Change this to be in the standard EV order for the UI input with correct mapping:
```suggestion
    def return_selection():
        nonlocal spinboxes, selected_ev_targets
        selected_ev_targets = StatsValues(
            hp=int(spinboxes[0].get()),
            attack=int(spinboxes[1].get()),
            defence=int(spinboxes[2].get()),
            special_attack=int(spinboxes[3].get()),
            special_defence=int(spinboxes[4].get()),
            speed=int(spinboxes[5].get()),
        )
        window.after(50, remove_window)
```